src/DynamoDbModel.php

```diff
@@ -195,10 +218,10 @@ public static function all($columns = [], $limit = -1)
         return $model->getAll($columns, $limit);
     }
 
-    public static function first($columns = [])
+    public function first($columns = [])
```

Comment:
Actually, if I recall correctly the same issue applies to chaining where calls. The first where works but any chained call will lose the state (due to static:getInstance()) and the result will contain all records for the second where (ignoring the first).